kmk/scanners/keypad.py

```diff
@@ -43,15 +43,26 @@ def __init__(
         *,
         columns_to_anodes=DiodeOrientation.COL2ROW,
         interval=0.02,
+        debounce_threshold=None,
         max_events=64,
     ):
-        self.keypad = keypad.KeyMatrix(
+        args = [
             row_pins,
             column_pins,
-            columns_to_anodes=(columns_to_anodes == DiodeOrientation.COL2ROW),
-            interval=interval,
-            max_events=max_events,
-        )
+        ]
+        for i in args:
+            if i is None:
+                args.pop(i)
+        kwargs = {
+            'columns_to_anodes': columns_to_anodes,
+            'interval': interval,
+            'debounce_threshold': debounce_threshold,
+            'max_events': max_events,
+        }
+        for key, value in kwargs.items():
+            if value is None:
+                kwargs.pop(key)
+        self.keypad = keypad.Keys(*args, **kwargs)
```

Comment:
oh dear. That is much too complicated for my taste. Do we want backwards compatibility with CP8?
1. No: none if this is necessary
2. Yes: Since we're not doing any validation nor any abstraction, just use pokemon arguments:
```
def __init__(self, *args, **kwargs):
    self.keypad = self.Keys(*args, **kwargs)
```
---
style sidenote: in any case, this screams comprehension:
```
args = [v for v in args if v is not None]
kwargs = {k:v for k,v in kwargs.items() if v is not None}
```

Reply:
I didn't go with comprehension because init had explicit arguments. I do like the pokemon arguments though, not sure why it wasn't that in the first place -- referring both to my PR and the main branch.